analysis.qmd

```diff
@@ -6,48 +6,59 @@ The draft currently has no official status. It is a work in progress and is subj
 
 # Analysis
 
-## Introduction
+## Introduction and summary
 
-The analysis phase is where planned analysis is undertaken, and progress and relevance are monitored. During work, the design and plan may be amended to account for changing circumstances, emerging information or unexpected difficulties or limitations encountered, and this phase also includes maintaining appropriate records of the analysis conducted, changes, decisions and assumptions made. In some cases, changes or limitations encountered may necessitate a return to either the design or scoping phase. 
+The analysis phase is where planned analysis is undertaken and assured, and progress and relevance are monitored. During this phase, the design may be amended to account for changing circumstances, emerging information or unexpected difficulties or limitations encountered. This phase also includes maintaining appropriate and traceable records of the analysis and assurance activities conducted, changes, decisions and assumptions made. In some cases, changes or limitations encountered may necessitate a return to either the design or scoping phase. 
 
-Throughout this phase, traceable documentation of the assurance activities undertaken shall also be produced. 
 
-In larger analytical projects, some outputs of the analysis may be completed at different times as work progresses, and aspects of other phases may therefore take place concurrently.
 
-## The Analyst's responsibilities during the analysis phase
+
+### The Analyst's responsibilities during the analysis phase
 
 As the analyst manages their analysis and follows their analytical plan, they must provide documentation of the data and methods used to the assurers. The analyst shall ensure these are sufficient for the assurer to understand the approach.
 
-When the analysts are following the analytical plan, they must record any changes to their plan if they encounter difficulties or unexpected limitations. The analyst will also carry out the assurance plan and shall provide traceable documentation of the assurance they have undertaken. They shall act on any findings from the assurer. 
+The analyst must record any changes to the analytical plan if they encounter difficulties or unexpected limitations. 
```

Comment:
Think the analysis recording changes in the plan should be a "should".

The analyst shall record changes to the analytical plan if they encounter difficulties or unexpected limitations. The final version of the plan shall reflect the analysis carried out. The aim being to enable the analysis to be repeated.